woocommerce/woocommerce-paypal-payments
# Capture virtual renewal orders functionality for subscription products

**Issue**: #626

---

### Description

When Intent is set to Authorize and Capture Virtual-Only is checked, subscription renewal orders do not change the order status to “Processing” or "Completed" if its a virtual+downloadable product.
The PR will add functionality to cover the behavior described above.

### Steps to Test




- On the PayPal Payment settings page, set "Payment Intent" to `Authorize` and check `Capture Virtual-Only Orders`
- Check out for a virtual subscription product
- The initial subscription order should be set to `Processing`
- The subscription will be active
- Process a renewal for  the subscription
- The renewal payment will be authorized but not captured automatically

---

Closes #626 .
